src/Services/ProductListDisplayService.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+
+require_once 'src/Entities/Product.php';
+
+class ProductListDisplayService
+{
+
+    public static function displayCategoryTitle(Product $productsInfo): string
+    {
+        return '<header class="container mx-auto md:w-2/3 md:mt-10 py-16 px-8 bg-slate-200 rounded">
+        <h1 class="text-5xl mb-2">Category: ' . $productsInfo->getCategoryName() . '</h1>
+        <p>For more information about any of the below products, click on the more button.</p>
+        </header>';
+    }
+
+    public static function displayProductList(Product $productsInfo): string
+    {
+        return '<section class="container mx-auto md:w-2/3 grid md:grid-cols-4 gap-5 mt-5">
```

Comment:
this puts each product into its own section. i think the section is supposed to surround all products with each one being a div. see the screenshot of how this looks in a browser
![image](https://github.com/iO-Academy/2024-mar-sde-furniture-store-woodworms/assets/6569497/e662c699-e2df-4ad7-8bd6-aac8e85efe5d)

Reply:
Done.